test/telephone_number/files/telephone_number_data_file_override.xml

```diff
@@ -0,0 +1,250 @@
+<!-- Copyright (C) 2009 The Libphonenumber Authors
```

Comment:
The more I think about this the more I think that the override file is just as contrived as what I would come up with. None of this stuff represents reality